docs/workflows/ios-specific-workflow-steps/xcode-select.md

```diff
@@ -57,6 +57,20 @@ You can review the documentation for detailed information about the Xcode versio
 
 This step contains some input variable(s). It needs these variable(s) to work. The table below gives explanation for this variable(s).
 
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE5421-xcodeSelectInput.png' />
+
+:::info Manually Xcode Version Change
+
+In addition to the version change method described above, you can also change the Xcode version **manually**. For this, you can **hard-code** the desired Xcode version into the `$AC_XCODE_VERSION` parameter. For example: `15.1`
```

Comment:
This usage is valid for all steps and all inputs. Why do we specify this only in Xcode?